cmd/test/test.go

```diff
@@ -0,0 +1,228 @@
+					if err != nil {
+						return fmt.Errorf("output results: %w", err)
+					}
+					fmt.Printf("%s\n", reportOutput)
+
+				} else {
+					// Conftest handles the output
+					outputter := output.Get(runner.Output, output.Options{
+						NoColor:            runner.NoColor,
+						SuppressExceptions: runner.SuppressExceptions,
+						Tracing:            runner.Trace,
+						JUnitHideMessage:   viper.GetBool("junit-hide-message"),
+					})
+					if err := outputter.Output(results); err != nil {
+						return fmt.Errorf("output results: %w", err)
+					}
+				}
+
+				// When the no-fail parameter is set, there is no need to figure out the error code
+				// as we always want to return zero.
+				if runner.NoFail {
+					return nil
+				}
+			}
+
+			os.Exit(exitCode)
+			return nil
+		},
+	}
+
+	cmd.Flags().Bool("fail-on-warn", false, "Return a non-zero exit code if warnings or errors are found")
+	cmd.Flags().Bool("no-fail", false, "Return an exit code of zero even if a policy fails")
+	cmd.Flags().Bool("no-color", false, "Disable color when printing")
+	cmd.Flags().Bool("suppress-exceptions", false, "Do not include exceptions in output")
+	cmd.Flags().Bool("all-namespaces", false, "Test policies found in all namespaces")
+	cmd.Flags().Bool("quiet", false, "Disable successful test output")
+
+	cmd.Flags().BoolP("trace", "", false, "Enable more verbose trace output for Rego queries")
+	cmd.Flags().BoolP("strict", "", false, "Enable strict mode for Rego policies")
+	cmd.Flags().BoolP("combine", "", false, "Combine all config files to be evaluated together")
+
+	cmd.Flags().String("ignore", "", "A regex pattern which can be used for ignoring paths")
+	cmd.Flags().String("parser", "", fmt.Sprintf("Parser to use to parse the configurations. Valid parsers: %s", parser.Parsers()))
+	cmd.Flags().String("capabilities", "", "Path to JSON file that can restrict opa functionality against a given policy. Default: all operations allowed")
+	validOutputs := append(output.Outputs(), OutputAppstudio)
+	cmd.Flags().StringP("output", "o", output.OutputStandard, fmt.Sprintf("Output format for conftest results - valid options are: %s", validOutputs))
+	cmd.Flags().Bool("junit-hide-message", false, "Do not include the violation message in the JUnit test name")
+
+	cmd.Flags().StringSliceP("policy", "p", []string{"policy"}, "Path to the Rego policy files directory")
+	cmd.Flags().StringSliceP("update", "u", []string{}, "A list of URLs can be provided to the update flag, which will download before the tests run")
+	cmd.Flags().StringSliceP("namespace", "n", []string{"main"}, "Test policies in a specific namespace")
+	cmd.Flags().StringSliceP("data", "d", []string{}, "A list of paths from which data for the rego policies will be recursively loaded")
+
+	cmd.Flags().StringSlice("proto-file-dirs", []string{}, "A list of directories containing Protocol Buffer definitions")
+
+	return &cmd
+}
+
+var TestCmd *cobra.Command
+
+func init() {
+	TestCmd = NewTestCommand()
+}
```

Comment:
These lines are not in the conftest version since they do the context handling slightly differently. We could tweak that maybe, but I don't think it's needed in this PR.